src/core/server/http/http_config.ts

```diff
@@ -21,7 +21,8 @@ import {
 } from './security_response_headers_config';
 
 const validBasePathRegex = /^\/.*[^\/]$/;
-const uuidRegexp = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+export const uuidRegexp =
```

Comment:
I considered moving this up to something like `server/common` but it didn't seem worth introducing it for this change.

Reply:
If we want to move this to somewhere more accessible, it would probably make sense to move it to a package such as `kbn/utils`. But I'm fine keeping this here for now.

FWIW, there's also a `validate` function from the `uuid` module we're using:

https://www.npmjs.com/package/uuid#uuidvalidatestr

It would probably make sense to rely on that instead of maintaining our own regexp, but that's a NIT (and outside of the scope of the PR).

Reply:
Indeed. Looks like we still have a very old version of `uuid` so first step would probably be upgrading it, then we could use the validator. I happened across https://github.com/elastic/kibana/blob/7dd29a56ad89dc26d2381a8e537b4b11a3930f27/packages/kbn-securitysolution-io-ts-types/src/uuid/index.ts#L12 which contains the same sort of validation.

Reply:
I opened up https://github.com/elastic/kibana/issues/123722 to cover the upgrade and validation replacement.